src/api/config.js

```diff
@@ -1,31 +1,53 @@
 'use strict'
 
-const argCommand = require('../cmd-helpers').argCommand
-
 module.exports = (send) => {
   return {
-    get: argCommand(send, 'config'),
-    set (key, value, opts, cb) {
-      if (typeof (opts) === 'function') {
-        cb = opts
+    get (key, callback) {
+      if (typeof key === 'function') {
+        callback = key
+        key = undefined
+      }
+
+      if (!key) {
+        return send('config/show', null, null, null, true, callback)
```

Comment:
💯 agree. In fact, what I'm very much thinking is after merging the config-api PR into ipfs-api/ng branch, just make a PR to update how request-api.js works, break the tests (so that we don't have to run 300 tests while developing a single feature) and clean up whatever else I find.